firmware/Core/Src/unit_tests/test_telecommand_arg_helpers.c

```diff
@@ -1,6 +1,35 @@
 #include "telecommands/telecommand_args_helpers.h"
 #include "unit_tests/unit_test_helpers.h"
 
+uint8_t TEST_EXEC__TCMD_ascii_to_uint64() {
+    uint64_t result;
+
+    // Success: Nominal test
+    TEST_ASSERT(TCMD_ascii_to_uint64("123", 3, &result) == 0);
+    TEST_ASSERT(result == 123);
+
+    // Error: Empty string
+    TEST_ASSERT(TCMD_ascii_to_uint64("", 0, &result) == 1);
+    TEST_ASSERT(result == 0);
+
+    // Error: String size doesn't match str_len
+    TEST_ASSERT(TCMD_ascii_to_uint64("123456", 10, &result) == 2);
+    TEST_ASSERT(result == 0);
```

Comment:
Please remove these `result == 0` on error; the result should be unchanged on error.